.gitignore

```diff
@@ -0,0 +1,17 @@
+node_modules/
+
+# Installed npm modules
+package-lock.json
```

Comment:
you have the package-lock.json checking in... I don't think we want it in the .gitignore